src/utils/Renderers.tsx

```diff
@@ -361,6 +365,38 @@ export const createWaypointsTableBody = (
   );
 };
 
+export const createVisibleRowCount = (
+  stateManager: Types.StateManager,
+): JSX.Element =>{
+  const[state] = stateManager;
+  const originalRows = state.allCombinedRows;
+  const filterPredicate = getFilterPredicate(state);
+  const rowCounterFiltered = originalRows.filter(filterPredicate);
+  const rowcounts = rowCounterFiltered.length;
+  totalInstanceCounter = state.rows.length;
+  currentInstancesCounter = 0;
+  
+  for(var j = 0; j < rowcounts; j++){
```

Comment:
You can replace `var` with `let`, be careful to use `var` since people will be confused if this value is mutable or not